README.md

```diff
@@ -79,10 +79,10 @@ This portfolio website built using HTML, CSS, Git and GitHub. It showcases  work
 
```

Comment:
- [ ] Make sure to customize your Readme file appropriately. Replace the place holder with your project name.

Reply:
Alright, Thank you for those feedback @Yerimah. Let me work on it